### ISSUE TITLE
Invalid mapping error occured when browserify's debug is true

### ISSUE THREAD

**Issue by rina-sleeping** [twada/licensify#17]
After I upgrade licensify version 2.0.0 to 3.1.0, Invalid mapping error occurred.
I tried to set browserify's debug option false, then  build succeed.
If browserify's debug option is true, this error always occur.
I can not reproduce this with simple files.
So I can write here only gulpfile file and version information.
Sorry if my mistake.

**Versions**
licensify: 3.1.0
browserify: 12.0.2

**gulpfile.babel.js**
```
gulp.task('scripts', () => {
  browserify('src/js/app.js', { debug: true})
    .transform('babelify', {
      presets: ['es2015', 'react'],
      plugins: ['transform-function-bind', 'transform-class-properties']
    })
    .plugin('licensify')
    .bundle()
    .on('error', err => { console.log('Error : ' + err.message); process.exit(-1); })
    .pipe(source('bundle.js'))
    .pipe(sourcemaps.init({ loadMaps: true }))
    .pipe(uglify({ preserveComments: 'license' }))
    .pipe(sourcemaps.write('./'))
    .pipe(gulp.dest('dist'));
});
```

**Error**
```
$ gulp scripts
[15:21:49] Failed to load external module babel-register
[15:21:49] Requiring external module babel-core/register
[15:21:51] Using gulpfile ~/apps/gulpfile.babel.js
[15:21:51] Starting 'html'...
[15:21:51] Finished 'html' after 13 ms
[15:21:51] Starting 'scripts'...
[15:21:51] Finished 'scripts' after 32 ms
[15:21:51] Starting 'build'...
[15:21:51] Finished 'build' after 5.16 μs
/home/app/node_modules/licensify/node_modules/offset-sourcemap-lines/node_modules/source-map/lib/source-map-generator.js:277
      throw new Error('Invalid mapping: ' + JSON.stringify({
      ^

Error: Invalid mapping: {"generated":{"line":41579,"column":3},"source":null,"original":{"line":null,"column":null},"name":null}
    at SourceMapGenerator_validateMapping [as _validateMapping] (/home/apps/node_modules/licensify/node_modules/offset-sourcemap-lines/node_modules/source-map/lib/source-map-generator.js:277:13)
    at SourceMapGenerator_addMapping [as addMapping] (/home/apps/node_modules/licensify/node_modules/offset-sourcemap-lines/node_modules/source-map/lib/source-map-generator.js:101:12)
    at /home/apps/node_modules/licensify/node_modules/offset-sourcemap-lines/index.js:22:19
    at Array.forEach (native)
    at SourceMapConsumer_eachMapping [as eachMapping] (/home/apps/node_modules/licensify/node_modules/offset-sourcemap-lines/node_modules/source-map/lib/source-map-consumer.js:155:14)
    at offsetLines (/home/apps/node_modules/licensify/node_modules/offset-sourcemap-lines/index.js:21:14)
    at DestroyableTransform._transform (/home/apps/node_modules/licensify/lib/licensify.js:240:37)
    at DestroyableTransform.Transform._read (/home/apps/node_modules/licensify/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/home/apps/node_modules/licensify/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/home/apps/node_modules/licensify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:313:64)
    at writeOrBuffer (/home/apps/node_modules/licensify/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:302:5)

```

**Comment by twada** [comment]
@rina-sleeping Thanks you for reporting! Sorry for inconvenience.

> If browserify's debug option is true, this error always occur.
I can not reproduce this with simple files.
So I can write here only gulpfile file and version information.


Okay, I'm going to investigate this issue with small React App.

**Comment by twada** [comment]
FYI: I've created [a small reproduction case](https://github.com/twada/licensify/issues/18) for this issue and going to fix it ASAP.

**Comment by twada** [comment]
@rina-sleeping Just released [licensify 3.1.1](https://github.com/twada/licensify/blob/master/CHANGELOG.md#311-2016-05-12). Would you try it?

**Comment by rina-sleeping** [comment]
It works well.
Thanks.
